analyzer/src/main/kotlin/managers/Conan.kt

```diff
@@ -272,8 +275,12 @@ class Conan(
     /**
      * Return the value `conan inspect` reports for the given [field].
      */
-    private fun runInspectRawField(pkgName: String, workingDir: File, field: String): String =
-        run(workingDir, "inspect", pkgName, "--raw", field).stdout
+    private fun inspectField(pkgName: String, workingDir: File, field: String): String =
+        pkgInspectResults.getOrPut(pkgName) {
+            val jsonFile = createOrtTempFile(managerName)
+            run(workingDir, "inspect", pkgName, "--json", jsonFile.absolutePath).requireSuccess()
+            jsonMapper.readTree(jsonFile).also { jsonFile.delete() }
+        }[field].textValueOrEmpty()
```

Comment:
Nit: IMO this mix of curly braces and square brackets does not read so nice. Maybe use `.get(field)` instead? (I've verified that the IDE does not hint to use `[]` in this case.)